.github/workflows/quality_check.yml

```diff
@@ -38,12 +38,14 @@ jobs:
     env:
       NODE_ENV: dev
       POWERTOOLS_DEV: true
+      GITHUB_TOKEN: ${{ secrets.PAT_GITHUB_BOT_USER }}
```

Comment:
suggestion: only expose this where we need it (L65), so that a dependency being installed (after install) doesn't have access to the secret in plain text.

Reply:
how would that look in the workflow? 

Do I do like `run: GITHUB_TOKEN=${{ secrets.PAT_GITHUB_BOT_USER }} npm test -- --silent --run` or can I set `env` in each step?

Reply:
Ok - I think I understood.

TIL: I didn't know you could add step-scoped `env`s.